_docs-sources/pipelines/upgrading/index.md

```diff
@@ -0,0 +1,166 @@
+# Upgrading from the ECS Deploy Runner
+
+This migration guide is intended for those running the ECS Deploy Runner (EDR) based Gruntwork Pipelines upgrading to Modern Gruntwork Pipelines. Upgrading can be done using Pipelines itself in a series of steps using code generation and pull requests.
```

Comment:
"Modern Gruntwork Pipelines" gets clunky. The first time we mention it up here we should say something like:

Modern Gruntwork Pipelines (Referred to as Gruntwork Pipelines or simply Pipelines in the rest of this article. The prior version will be explicitly called EDR pipelines for the remainder of this article.)

I don't feel super strongly about this though, so pls ignore if you want, I just got tired of reading extra words.

Reply:
Good call, it used to say `v2` then i did a find a replace. Your idea is better!

Reply:
Fixed in [73295b2](https://github.com/gruntwork-io/docs/pull/1115/commits/73295b26e2fbe7ab38f0addf4381fb0d3a67be12)